docs/reference/query-rules/apis/delete-query-rule.asciidoc

```diff
@@ -0,0 +1,74 @@
+[role="xpack"]
+[[delete-query-rule]]
+=== Delete query rule
+
+++++
+<titleabbrev>Delete query rule</titleabbrev>
+++++
+
+Removes an individual query rule within an existing query ruleset.
+This is a destructive action that is only recoverable by re-adding the same rule via the <<put-query-rule, PUT query rule>> API.
```

Comment:
I'd say "Create or update query rule" instead of `PUT query rule` if possible as it'd make it more declarative for the action than just "PUT"...  (I can't remember if asciidoc will allow that though ;) ) -